src/commands/merge.ts

```diff
@@ -0,0 +1,55 @@
+import { commands, window } from "vscode";
+import { IBranchItem } from "../common/types";
+import { isTrunk, selectBranch } from "../helpers/branch";
+import { Repository } from "../repository";
+import { Command } from "./command";
+
+export class Merge extends Command {
+  constructor() {
+    super("svn.merge", { repository: true });
+  }
+
+  public async execute(repository: Repository) {
+    const branch = await selectBranch(repository);
+
+    if (!branch) {
+      return;
+    }
+
+    await this.merge(repository, branch);
+  }
+
+  async merge(repository: Repository, branch: IBranchItem) {
+    let reintegrate = false;
+    if (isTrunk(repository.currentBranch)) {
+      reintegrate = true;
+    }
+
+    try {
+      await repository.merge(branch.path, reintegrate);
+    } catch (error) {
+      console.log(error);
```

Comment:
Remove this

Reply:
Sorry I don't get what you propose to remove. The entire try catch? or simply the log of the error? I guess the second, right? How about moving it to the last else block l.51, as there one has no clue what went wrong?

Reply:
Then a log of the error could be useful.

Reply:
Yeah moving the console.log to the else block would be okay